src/launcher/features/apps/appDialogsSlice.test.ts

```diff
@@ -29,8 +29,11 @@ describe('app dialogs slice', () => {
 
             const dialogIsShown = dispatchTo(reducer, [
                 showConfirmLaunchDialog({
-                    text: 'Do you confirm?',
                     app,
+                    title: 'Please',
```

Comment:
Is this test line mirrored somewhere in the user-facing UI?

Reply:
Yes, this is only data within the tests and nothing that appears on screen.